homeassistant/components/smtp/notify.py

```diff
@@ -52,53 +47,41 @@
     DEFAULT_PORT,
     DEFAULT_TIMEOUT,
     DOMAIN,
-    ENCRYPTION_OPTIONS,
 )
 
-PLATFORMS = [Platform.NOTIFY]
-
 _LOGGER = logging.getLogger(__name__)
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Don't remove the schema until we remove the import.

Reply:
Addressed in #110857